pyproject.toml

```
@@ -1,5 +1,5 @@
 [tool.black]
-line-length = 79
+line-length = 115
```

keep it with 79 pls, as specified by PEP8

```suggestion
line-length = 79
```